Cargo.toml

```diff
@@ -54,7 +54,7 @@ uuid = { version = "1.4.1", features = ["v4", "fast-rng", "macro-diagnostics", "
 wsl = { version = "0.1.0" }
 
 # v1.0.1
-libcst = { git = "https://github.com/Instagram/LibCST.git", rev = "9c263aa8977962a870ce2770d2aa18ee0dacb344", default-features = false }
+libcst = { version = "0.1.0" }
```

Comment:
The comment is outdated as I forgot to update it when I updated LibCST to the latest commit in https://github.com/astral-sh/ruff/pull/7062 (Sorry!). So, we'll have to wait until there's another release for us to update here.

Reply:
Or am I mistaken? Does `0.1.0` indicate the latest version of main as of now i.e., does it include this commit (instagram/libcst@9c263aa8977962a870ce2770d2aa18ee0dacb344)? I think it does as the tests seems to be passing.

Reply:
It seems to be `main`. Or at least any commit not older than https://github.com/Instagram/LibCST/commit/377a292d0d43a71b1e9704799d7e46b621a97018 where they added the metadata. But I agree, the versioning is a bit confusing